src/components/login.css

```diff
@@ -0,0 +1,26 @@
+div#login{
+    width: 100%;
+    border-radius: 10px;
+}
+
+div#login > form {
+    border: 6px solid cyan;
+    border-radius: 12px;
+    padding: 4rem;
+    margin: 2rem 30rem;
+}
+
+h1{
+    text-align: center;
+}
+
+#login button[type="submit"] {
+    width: 50%;
+    height: 50px;
+    border-radius: 14px;
+    background-color: chartreuse;
+}
+
+#login {
```

Comment:
does the h1 need `text-align: center` if the parent element has `text-align: center`? If you do need to keep text-align for `#login`, then please group it with the `div#login` selector at the top